src/node.cc

```diff
@@ -4803,18 +4803,20 @@ inline static bool TickEventLoop(Environment & env) {
   bool more = false;
   uv_run(env.event_loop(), UV_RUN_NOWAIT);
 
+  if (uv_loop_alive(env.event_loop())) {
+    return true;
+  }
+
   v8_platform.DrainVMTasks();
 
-  more = uv_loop_alive(env.event_loop());
-  if (more)
-    return more;
+  if (uv_loop_alive(env.event_loop()))
```

Comment:
We should either add curly braces around the return statement here or remove them in line 4806 to be consistent.